### PR TITLE
Add measures for custom commands

### PR DESCRIPTION
**Pull request by edrevo**
### What and why?

Add measure support to custom commands. Fixes https://github.com/DataDog/datadog-ci/issues/1261

### How?

Send new field to backend.

### Review checklist

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/trace/trace.ts [hunk 1]
```diff
@@ -101,6 +104,7 @@ export class TraceCommand extends Command {
           error_message: stderr,
           exit_code: exitCode,
           is_error: exitCode !== 0,
+          measures,
```
Comment (juan-fernandez) [inline src/commands/trace/trace.ts:107]: are we OK with this sending null measures? If the user sets a string measure `parseFloat` would return `NaN` which is serialized as `null` by `JSON.stringify`: 
```
> JSON.stringify({ a: parseFloat('a')})
'{"a":null}'
```

**Review by juan-fernandez — APPROVED** [review]

File: src/commands/trace/trace.ts (hunk 1 above)
Comment (edrevo) [reply src/commands/trace/trace.ts:107]: I tried that and it ends up as a 0 after all the processing. I'd say it's an ok-ish behavior, but maybe it's better to fail the command? What do you think?

**Review by Drarig29 — APPROVED** [review]
LGTM

File: src/commands/trace/trace.ts (hunk 1 above)
Comment (juan-fernandez) [reply src/commands/trace/trace.ts:107]: I don't think it's worth failing the command, but a `0` doesn't seem correct either. Maybe we can do: 

```javascript
Object.fromEntries(
    Object.entries(cliMeasures)
        .map(([key, value]) => [key, parseFloat(value)])
        .filter(([key, value]) => {
            if (isNaN(value)) {
                logger.debug(`Value for ${key} is not a number`);
                return false;
            }
            return true;
        }),
);
```
WDYT? Maybe even without the logger.

**Review by juan-fernandez — APPROVED** [review]

**Pull request merged**